components/x-teaser/stories/index.js

```diff
@@ -1,5 +1,9 @@
 const { Teaser } = require('../');
 
+if (window && !/\.ft\.com$/.test(window.location.hostname)) {
```

Comment:
`ReferenceError: window is not defined`

goes to show how careful we need to be about assuming environments 😬

Reply:
Strange it didn't blow up my local build 🤷‍♂️ 

Easy to fix at least